PythonTool/PlotPseudocolor.py

```diff
@@ -0,0 +1,51 @@
+from visit import *
+
+
+def PlotPseudocolor(File, ObjectSequence):
+    # Pseudocolor plot attributes.
+
+    # Change orientation of legend
+    LegendAttributes = GetAnnotationObject(ObjectSequence["Pseudocolor"])
+    LegendAttributes.orientation = "VerticalLeft"
+    LegendAttributes.fontHeight = 0.03
+
+    Attribute = PseudocolorAttributes()
+
+    if File is not None:
+        File = list(File)
+
+        for item in File:
+            if item[1].title() == "Pseudocolor":
+
+                # Check for optional inputs in Dictionary input.
+                try:
+                    Attribute.scaling = eval(
+                                             "Attribute." +
+                                             str(item[3]).title()
+                                             )
+                except Exception:
+                    pass
+
+                try:
+
+                    if (item[4])[0].title() == "Min":
```

Comment:
Comments on why you're using `item[4]` here would be helpful. Dictionaries sometimes provide a way of indicating what a value is via keys rather than requiring one to know what each index in a list is mean to represent throughout the code.